Update and Fix (#13)
wso2/docs-ob-toolkit-cds: en/docs/get-started/configure-sms-otp-authenticator.md

> This document provides step by step instructions to setup SMS OTP configurations

Reviewer: Add full stop

## Sign in to the Identity Server

> 1. Sign in to the Management Console of WSO2 Identity Server at [https://localhost:9446/carbon](https://localhost:9446/carbon)

Reviewer: Let's add full stop here as well

2. Use the default super admin credentials as follows:
    - Username: admin@wso2.com
    - Password: wso2123

    !!!note
        The above login credentials are for testing purposes only. It is recommended to change the login credentials in
        a production environment.

### Step 1: Adding SMS OTP Identity Provider

> 1. Go to the **Main menu** -> **Identity** -> **Identity Providers** -> **Add**.

Reviewer: ```suggestion
1. Go to the **Main menu > Identity > Identity Providers > Add**.
```

Reviewer: shall we change as above for consistency. That is how we used paths in other pages :)
Pls check all the places in this doc and its duplicate

      ![add_identity_providers](../assets/img/get-started/quick-start-guide/go-to-add-identity-providers.png)

> 2. Fill the **Basic Information** section and name this identity provider as **SMSAuthentication**

Reviewer: ```suggestion
2. Fill the **Basic Information** section and name this Identity Provider as **SMSAuthentication**
```

      ![sms_authentication_idp](../assets/img/get-started/quick-start-guide/sms-authentication-idp.png)

> 3. Expand the **Federated Authenticators** -> **SMS OTP Configuration** section

Reviewer: Let's add full stop where we have used complete sentences. 
Pls check all the places in this doc and its duplicate

      ![security_account_lock](../assets/img/get-started/quick-start-guide/login-security-account-lock.png)

2. Select the **Lock User Accounts** checkbox

      ![maximum_login_attempts](../assets/img/get-started/quick-start-guide/maximum-failed-login-attempts.png)

3. Configure the **Maximum Failed Login Attempts** 

4. Scroll down and click **Update**

### Step 3: Making Mobile Phone a Mandatory Claim

1. Go to the **Main menu** -> **Identity** -> **Claims** -> **List** -> **http://wso2.org/claims** -> **Mobile** -> **Edit**

      ![select_claims_lists](../assets/img/get-started/quick-start-guide/go-to-claims-lists.png)

      ![edit_mobile_claim](../assets/img/get-started/quick-start-guide/edit-mobile-claim.png)

2. Select the **Required** checkbox

      ![update_claim_details](../assets/img/get-started/quick-start-guide/update-local-claim-details.png)

3. Scroll down and click **Update**

!!! tip "More on Login Information..."
     In the authentication flow, if you login as an admin user, it will prompt for the mobile number in the first
     attempt to login. Mobile number should be given in the format of the following example - 94714564567

!!!note 

>      Test  scenarios can include attempting to login using invalid usernames more than allowed number of times, attempting

Reviewer: ```suggestion
     Test  scenarios can include attempting to log in using invalid usernames more than the allowed number of times, attempting
```

>      invalid OTPs more than allowed number of times, etc.

Reviewer: ```suggestion
     invalid OTPs more than the allowed number of times, etc.
```
